### PR TITLE
Removed union class equivalence assertion from gist:Artifact

### LINKED ISSUE #110: Remove equivalence to a union class from Artifact definition
When a class is defined as the union of a set of other classes, it is not possible to define additional subclasses. Is this by intention? It makes extension difficult. 

For example:

```
	<owl:Class rdf:about="&gist;Artifact">
		<rdfs:label rdf:datatype="&xs;string">Artifact</rdfs:label>
		<rdfs:comment rdf:datatype="&xs;string">An intentional person made thing, could be physical or content</rdfs:comment>
		<owl:intersectionOf rdf:parseType="Collection">
			<owl:Class>
				<owl:unionOf rdf:parseType="Collection">
					<rdf:Description rdf:about="&gist;Content">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;IntellectualProperty">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;Equipment">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;Building">
					</rdf:Description>
					<rdf:Description rdf:about="&gist;PhysicalSubstance">
					</rdf:Description>
				</owl:unionOf>
			</owl:Class>
			<owl:Restriction>
				<owl:onProperty rdf:resource="&gist;goal"/>
				<owl:someValuesFrom rdf:resource="&gist;Function"/>
			</owl:Restriction>
		</owl:intersectionOf>
	</owl:Class>
```
Because this is a closed list, an ontology importing gist cannot define another direct subclass of Artifact.

In fact, we see already this problem within gist itself. System is a subclass of Artifact, but it is not asserted to subclass any of the unioned classes. Do we really want to say that System is a direct subclass of one of these, but we don't know which? Seems odd. And none of the unioned classes seem like appropriate superclasses of System.

### PR DESCRIPTION
**Pull request by rjyounes**
Fixes #110.

@marksem Added a release note to encompass both this issue and #343 for Place.

### CODE REVIEW

File: gistCore.ttl [hunk 1]
```diff
@@ -195,7 +180,12 @@ gist:Behavior
 
 gist:Building
 	a owl:Class ;
-	rdfs:subClassOf gist:Landmark ;
+	rdfs:subClassOf [
+		owl:intersectionOf (
+			gist:Landmark
+			gist:Artifact
+		) ;
+	] ;
```
Comment (uscholdm) [inline gistCore.ttl:188]: I would have expected to see:
```
gist:Building
	a owl:Class ;		
	rdfs:subClassOf gist:Landmark , gist:Artifact .
```
I think the meaning is different, but am not 100% sure.

File: gistCore.ttl [hunk 2]
```diff
@@ -890,8 +881,8 @@ gist:Equipment
 	rdfs:label "Equipment"^^xsd:string ;
 	rdfs:comment "Tangible property other than land or buildings.  Any kind of equipment, could be machine, router, car etc."^^xsd:string ;
 	owl:equivalentClass [
-		a owl:Class ;
```
Comment (uscholdm) [inline gistCore.ttl:893]: Did the serializer remove this?

File: gistCore.ttl [hunk 3]
```diff
@@ -1946,18 +1938,21 @@ gist:PhysicalIdentifiableItem
...
-		] ,
-		[
-			a owl:Restriction ;
-			owl:onProperty gist:hasMagnitude ;
-			owl:someValuesFrom gist:Volume ;
-		]
-		;
+	rdfs:subClassOf [
+		owl:intersectionOf (
+			gist:Artifact
```
Comment (uscholdm) [inline gistCore.ttl:1943]: This is a mistake that was already in gist.  The lump of dirt I have in my hand is not an Artifact.  It should be removed.

File: gistCore.ttl (hunk 2 above)
Comment (rjyounes) [reply gistCore.ttl:893]: I removed it - note that it's redundant, given the range of owl:equivalentClass. However, as above I'll add it back so we can push this through.

**Comment by rjyounes** [comment]
I wrote one release note for both, so the language will be the same.

File: gistCore.ttl (hunk 1 above)
Comment (rjyounes) [reply gistCore.ttl:188]: The meaning is the same. Your way says everything that's a Building is both a Landmark and an Artifact. My way says that everything that's a Building lies at the intersection of Landmark and Artifact. These are logically identical. That said, I can do it either way, so I'll switch it to get this through more quickly.

Comment (rjyounes) [reply gistCore.ttl:188]: Yours is stylistically simpler and probably more comprehensible, as well.

Comment (uscholdm) [reply gistCore.ttl]: Your description made the equivalence clear - my brain must have been muddled!

**Comment by uscholdm** [comment]
The definition of GeoRoute has OrderedCollection twice. Suggest to remove it from the subclass predicate. 
```
gist:GeoRoute
	a owl:Class ;
	rdfs:subClassOf
		gist:OrderedCollection ,   # THIS IS REDUNDANT
		gist:Place
		;
	rdfs:label "Geo Route"^^xsd:string ;
	rdfs:comment "An ordered set of GeoPoints that defines a path from starting point to ending point."^^xsd:string ;
	owl:equivalentClass [
		a owl:Class ;
		owl:intersectionOf (
			gist:OrderedCollection
			[
				a owl:Restriction ;
				owl:onProperty gist:hasDirectPart ;
				owl:someValuesFrom gist:GeoSegment ;
			]
		) ;
	] ;
	.
```

**Comment by rjyounes** [comment]
@uscholdm Is the GeoRoute issue relevant here? If not, I suggest creating a new issue. I think it's more straightforward if we remove the subclass assertion altogether:
```
gist:GeoRoute
	a owl:Class ;
         # REMOVE ENTIRE SUBCLASS STATEMENT
	#rdfs:subClassOf
	#	gist:OrderedCollection ,   # THIS IS REDUNDANT
	#	gist:Place
	#	;
	rdfs:label "Geo Route"^^xsd:string ;
	rdfs:comment "An ordered set of GeoPoints that defines a path from starting point to ending point."^^xsd:string ;
	owl:equivalentClass [
		a owl:Class ;
		owl:intersectionOf (
			gist:OrderedCollection

			[
				a owl:Restriction ;
				owl:onProperty gist:hasDirectPart ;
				owl:someValuesFrom gist:GeoSegment ;
			]
		) ;
	] ;
	.
```

**Comment by uscholdm** [comment]
>  I think it's more straightforward if we remove the subclass assertion altogether:

@rjyounes Yes, that is a perfectly and logically equivalent alternative go for it. 

Why is this closed? The work was pretty much done!!

**Comment by rjyounes** [comment]
I'm not sure how it got closed. I've reopened now and will merge.

**Comment by rjyounes** [comment]
Created new issue #361 for the GeoRoute fix.

**Pull request merged**